docs/build/workspace.md

```diff
@@ -0,0 +1,88 @@
+# Tutorial: Integrating multiple packages in a workspace
+
+In this tutorial, we will show you how to integrate multiple pixi packages into a single workspace.
+
+## Why is this useful?
+
+The packages coming from conda channels are already built and ready to use.
+If you want to depend on a package you therefore typically get that package from such a channel.
+However, there are situations where you want to depend on the source of a package.
```

Comment:
maybe `source code` of a package? Like, I think it is more common to hear it in this way